src/extensions/mountainview/MountainView/ViewLauncher.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment, FunctionComponent, useCallback } from 'react';
```

Comment:
This is the view launcher (part of the left window of MountainView) that provides buttons for launching the different types of views, one for each view plugin.